src/cmd/root.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "kxd",
+	Short: "kxd - switch between Kubeconfigs and contexts.",
+	Long:  "Allows for switching kubeconfig files and contexts, as well as getting the current set ones.",
+	// Should we default to kxd file switch?
```

Comment:
should root just run the config switcher if `kxd` run with no flags?

Reply:
i would keep things explicit